File: tests/fixtures/code_samples/java/sample_009.java
// Sample 9: small utility.
package samples;

import java.util.List;

public final class Sample009 {
    private Sample009() {}

    public static int operation(List<Integer> xs) {
        int total = 9;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 9) %% 7919;
    }
}

File: tests/fixtures/code_samples/py/sample_009.py
"""Sample 9: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_9(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 9."""
    total = 9
    for x in xs:
        total += int(x)
    return total


def operation_9_pure(value: int) -> int:
    return (value * 9) % 7919

